### PR TITLE
Jewelry postapoc price changes

### PR DESCRIPTION
**Pull request by faefux**
#### Summary
```SUMMARY: Balance "Changed postapoc prices for various Jewelry pieces"```

#### Purpose of change
Pieces of jewelry were going for far to much, considering they are just old pieces of scrap metal at this point.
#### Describe the solution
gave most jewelry pieces a "price_postapoc".

#### Describe alternatives you've considered
Changing there base price or drop rate.

#### Additional context
None that i can think of. Feel free to recommend other prices if these arn't up to par.

### CODE REVIEW

File: data/json/items/armor.json [hunk 1]
```diff
@@ -7399,6 +7410,7 @@
     "weight": 30,
     "volume": 0,
     "price": 590000,
+    "price_postapoc": 59000,
```
Comment (anothersimulacrum) [inline data/json/items/armor.json]: That's 590 dollars, which is very different than the rest of your other changes.

**Comment by Regularitee** [comment]
So, umm, https://github.com/CleverRaven/Cataclysm-DDA/pull/31843

File: data/json/items/armor.json (hunk 1 above)
Comment (faefux) [reply data/json/items/armor.json]: Weird, don't even remember changing that one! :s oh well, made it about 20USD.

**Comment by faefux** [comment]
> So, umm, #31843

Changed the postapoc prices, that seems to be changing the preapoc prices, either way they're still real high for how useless and common they are.

**Comment by Regularitee** [comment]
> they're still real high for how useless and common they are.

My rationale is as follows:
- Many of your prices fail to account for practical uses. For example, copper and gold have value as a conductor -- and in a setting very heavily immersed in robotics and cybernetics, I find it hard to believe it'd sell for less than a literal piece of string. Especially given how much more common cloth and string is compared to refined copper.
- Likewise, watches have very low interchangeability of parts between units (as opposed to bullet calibers, or car parts), so would be very difficult to maintain. Especially given high precision required to manufacture miniaturized parts may not even be available anymore. And given how vital they are to survival (precisely predicting how long until night falls, blinding survivors without NVGs), they should probably be way up there in price.
- The current setting is past the "dog-eat-dog" stage of chaos, and society is beginning to rebuild (various factions have organized society). Even the player themselves have started rebooting society by way of the base camp. So luxury prices would not be at rock-bottom stage as they would be during a time of total anarchy.

And they do change post-apoc prices, as that was requested.

**Comment by anothersimulacrum** [comment]
Could you provide a list of the items you changed, and what they are worth with and without the new price_postapoc?  
It's not very easy to see in the changes view.

**Comment by faefux** [comment]
Hmmm, well okay. I see your points but only partially agree.
I'll leave this up to see what others think. the consensus on the discord seems to have been these  low prices.

**Comment by Regularitee** [comment]
Look, I'm really sorry to step on your toes like this. There wasn't much I could do since you have me blocked on discord (for saying "beggars can't be choosers" the other day), and since your discord name doesn't match your github name, there was no way let you know it was already being worked on until **after** you posted your PR.

Unfortunately, when another developer on the same project is invisible to you, odds are you'll end up eventually creating something that conflicts or overlaps with their work, eventually. (And it can't make discord discussion easy either, since when others reply to what I said, you end up thinking they're replying to what you said, since your view of chat logs don't sync up with everyone else's.)

**Comment by faefux** [comment]
> Could you provide a list of the items you changed, and what they are worth with and without the new price_postapoc?
> It's not very easy to see in the changes view.
You're right, it really isn't. Luckily i started to add these to there own file for my next PR.

`"id": "collarpin"
"name": "collar pin"
"price": 25000

"id": "copper_bracelet"
"name": "copper bracelet"
"price": 2500
"price_postapoc": 100

"id": "copper_ear"
"name": "copper earring"
"price": 2900
"price_postapoc": 50

"id": "silver_bracelet"
"name": "silver bracelet"
"price": 25000
"price_postapoc": 500

"id": "locket"
"name": "locket"
"price": 4000
"price_postapoc": 200
  
"id": "silver_ear"
"name": "silver earring"
"price": 8000
"price_postapoc": 200
  
"id": "gold_bracelet"
"name": "gold bracelet"
"price": 46000
"price_postapoc": 1000

"id": "gold_dental_grill"
"name": "gold dental grill"
"price": 20000
"price_postapoc": 500

"id": "gold_ear"
"name": "gold earring"
"price": 13000
"price_postapoc": 500

"id": "gold_watch"
"name": "gold watch"
"price": 50000
"price_postapoc": 2000

"id": "tieclip"
"name": "tie clip"
"price": 28000
"price_postapoc": 200

"id": "pearl_collar"
"name": "pearl collar"
"price": 590000
"price_postapoc": 2000

"id": "ring"
"name": "gold ring"
"price": 30000
"price_postapoc": 2000

"id": "brooch"
"name": "jade brooch"
"price": 90000
"price_postapoc": 400

  
"abstract": "badge_abstract"
"name": "badge template"
"price": 5000
"price_postapoc": 700

 
"id": "badge_detective"
"copy-from": "badge_abstract"
"name": "detective badge"
"price_postapoc": 2000

"id": "badge_doctor"
"copy-from": "badge_abstract"
"name": "doctor badge"
"price_postapoc": 30

"id": "badge_foodkid"
"copy-from": "badge_abstract"
"name": "Foodkid badge"
"price_postapoc": 30

"id": "diamond_dental_grill"
"name": "diamond dental grill"
"price": 50000
"price_postapoc": 1500

"id": "diamond_ring"
"name": "diamond ring"
"price": 190000
"price_postapoc": 1700


"id": "wristwatch"
"name": "wrist watch"
"price": 5000
"price_postapoc": 300
"coverage": 5
  
"id": "diving_watch"
"name": "diver's watch"
"price": 12000
"price_postapoc": 700

"id": "ear_spool"
"name": "plug"
"price": 1200
"price_postapoc": 10

"id": "holy_symbol"
"name": "holy symbol"
"price": 5000
"price_postapoc": 500


"id": "necklace"
"name": "silver necklace"
"price": 12000
"price_postapoc": 400

"id": "sf_watch"
"name": "fancy watch"
"price": 2200000
"price_postapoc": 3000

`


That should be right!!

**Comment by kevingranade** [comment]
Seems about right, I think there's a bit of over-valuation still, but it's in a very manageable range, and I'm not at all sure my valuation isn't underestimating how much people would want to squirrel away gold and diamonds, so I'd rather go with this than try and nit-pick our way to a more precise valuation.

**Comment by Regularitee** [comment]
@kevingranade What about the discrepancy between Hub 01 gold coins and gold jewelry? Given the gap in value between gold hub coins and gold jewelry, it looks like a lot of people could get rich buying jewelry and smelting them counterfeit coins (or not even counterfeit, since they'd be real gold coins).

**Comment by kevingranade** [comment]
There's a recipe for minting hub coins?

File: data/json/items/armor.json [hunk 2]
```diff
@@ -836,6 +836,7 @@
     "weight": 60,
     "volume": 0,
     "price": 5000,
+    "price_postapoc": 700,
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: I'm a little late with this, but this addition was unnecessary as you shouldn't find this in a normal game, ever.

**Comment by mlangsdorf** [comment]
Hub 01 coins are not just chunks of gold, they're chunks of gold with a fairly low pitch (130 nm or so) circuitry design etched in them.  You can't easily counterfeit them without access to a silicon fabrication facility - the design is going to look wrong if you try to do it by hand.

**Comment by I-am-Erk** [comment]
What would probably work well is to make NPCs with the Stylish trait pay a higher price for jewelry in trade... And eventually to work on a larger preferences and demand/supply system of course. In the probably long intervening timeframe this makes sense

**Pull request merged**